.github/workflows/build.yml

```diff
@@ -33,5 +33,6 @@ jobs:
             go.sum
             *.toml
       - run: |
-          make build
+          make build-docker-goleveldb
+          make build-docker-pebbledb
```

Comment:
FYI this is just a workflow to check that the application compiles successfully.
It is good anyways to make sure that the docker image can build without issues, but this action will only check that.